### DISCUSSION TITLE
Donation issues

### DISCUSSION THREAD

**Discussion by fine79** [aclap-dev/video-downloadhelper#400] [Help and Q&A]
### Your question

Hi there. Don't know this is 100% the right forum, but didn't quite know where else to raise this. I tried the donation page a few times and it kept getting declined. 

This was with one of those Vanilla brand giftcards, but it _is_ a mastercard and I believe the idea is it should function like a normal debit/credit. While it seems most likely it's an issue with Vanilla company, I did call them up and they said they saw the decline transactions, but claimed that my card was active and it wasn't something they were doing on their end. Just since I saw some of the mentioned previous issues with Paypal in the past, I thought I might raise the heads up in case it's something on the Stripe end. Mostly just FYI.

Thanks.

### Addon & CoApp version and details

N/A

**Comment by jcfrog** [comment]
Hi,
Thanks for the feedback. We do not manage the payments. Stripe does. I can only recommend to contact their support.
